awips_install.sh

```diff
@@ -386,6 +386,17 @@ function cave_prep {
  rm -rf /home/awips/caveData 
 }
 
+function cleanup {
+  sed -i 's/enabled=1/enabled=0/' /etc/yum.repos.d/awips2.repo
+  if $1; then
+    sed -i 's/@LDM_PORT@/388/' /awips2/ldm/etc/registry.xml 
+  fi
+  if $2; then
+    disable_ndm_update
+  fi
+  echo "$3 has finished installing, the install log can be found in /tmp/awips-install.log"
+}
+
```

Comment:
Either at this line or between 404 and 405, add this:
`alterReg=true`
`disableNDM=true`

Then only override them to false in the specific cases you need to.

Also, I think you could actually use the variable names $alterReg, $disalbeNDM in the cleanup function, since they've been defined before this function was called.  So you could just use "cleanup" at line 440, then use the actual variable names in the function which makes it much clearer/easier to read as well.